meson_options.txt

```diff
@@ -195,6 +195,8 @@ option('default-kill-user-processes', type : 'boolean',
        description : 'the default value for KillUserProcesses= setting')
 option('gshadow', type : 'boolean',
        description : 'support for shadow group')
+option('default-locale', type : 'string', value : 'C',
```

Comment:
please default to C.UTF-8 here.

Reply:
I have no interest in defaulting to a locale that does not exist. Anyways, yuwata's patch provides a better solution here.

Reply:
Sane defaults, by the very definition of the phrase, are things that work everywhere.